tracex_project/trace_comparator/comparator.py

```diff
@@ -82,70 +92,90 @@ def find_activity_mapping(view, pipeline_activities, ground_truth_activities):
 
 def compare_activities(
     view,
-    current_step,
-    total_steps,
-    status,
-    input_activities,
-    comparison_basis_activities,
-):
-    """Compare input activities with comparison basis activities."""
-    mapping_input_to_comparison = []
+    current_step: int,
+    total_steps: int,
+    status: str,
+    input_activities: pd.Series,
+    ground_truth_activities: pd.Series,
```

Comment:
I'd like to argue against calling this variable `ground_truth_activities` as this could also be the `pipeline_activities` (see function `mapping_ground_truth_to_pipeline()`). That's also why the previous variable was called `input_activities` instead of `pipeline_activities`. 
We use this function in both directions, which is why I think `input_activities` and `comparison_basis_activities` are decent names.